test/chartmogul/plan-group.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const ChartMogul = require('../../lib/chartmogul');
+const config = new ChartMogul.Config('token', 'secret');
+const expect = require('chai').expect;
+const nock = require('nock');
+const PlanGroup = ChartMogul.PlanGroup;
+
+describe('PlanGroup', () => {
+  it('should create a new plan group', () => {
+    /* eslint-disable camelcase */
+    const postBody = {
+      name: 'Bronze Plan Group',
+      plans: ['pl_eed05d54-75b4-431b-adb2-eb6b9e543206', 'pl_ab225d54-7ab4-421b-cdb2-eb6b9e553462']
+    };
+    /* eslint-enable camelcase */
+
+    nock(config.API_BASE)
+      .post('/v1/plan_groups', postBody)
+      .reply(200, {
+        /* eslint-disable camelcase */
+        name: 'Bronze Plan Group',
+        uuid: 'plg_eed05d54-75b4-431b-adb2-eb6b9e543206',
+        plans: [
+          {
+            uuid: 'pl_eed05d54-75b4-431b-adb2-eb6b9e543206',
+            external_id: 'plan_0001',
+            name: 'Bronze Plan',
+            interval_count: 1,
+            interval_unit: 'month',
+            data_source_uuid: 'ds_e243129a-12c0-4e29-8f54-07da7905fbd1'
+          },
+          {
+            uuid: 'pl_ab225d54-7ab4-421b-cdb2-eb6b9e553462',
+            external_id: 'plan_0001',
+            name: 'Bronze Plan',
+            interval_count: 1,
+            interval_unit: 'month',
+            data_source_uuid: 'ds_e243129a-12c0-4e29-8f54-07da7905fbd1'
+          }]
+        /* eslint-enable camelcase */
+      });
+
+    return PlanGroup.create(config, postBody)
+    .then(res => {
+      expect(res).to.have.property('uuid');
+      expect(res).to.have.property('plans');
+      expect(res.plans).to.be.instanceof(Array);
+    });
+  });
+
+  it('should get all plan groups', () => {
+    nock(config.API_BASE)
+    .get('/v1/plan_groups')
+    .reply(200, {
+      /* eslint-disable camelcase */
+      plan_groups: [],
+      current_page: 1,
+      total_pages: 0
+      /* eslint-enable camelcase */
+    });
+
+    return PlanGroup.all(config)
+    .then(res => {
+      expect(res).to.have.property('plan_groups');
+      expect(res.plan_groups).to.be.instanceof(Array);
+    });
+  });
+
+  it('should modify a plan group', () => {
```

Comment:
What do I do if I want to add/remove plans from groups?

Reply:
I'll add the specs 💃